src/wallet/wallet.h

```diff
@@ -723,6 +723,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      */
     mutable CCriticalSection cs_wallet;
 
+    std::atomic<bool> fAbortRescan;
```

Comment:
For some reason I didn't think we used getters/setters but clearly we do. Fixed, thanks!